admin/src/app/app.component.html

```diff
@@ -1 +1,7 @@
-<router-outlet></router-outlet>
\ No newline at end of file
+<app-sidebar
+  [isLoggedIn]="isLoggedIn"
+>
+  <router-outlet></router-outlet>
```

Comment:
sidebar should not get isLoggedIn input, show/hide disebar component using if not by passing input